lib/wos_queries.rb

```diff
@@ -0,0 +1,184 @@
+
+# Queries on the Web of Science (or Web of Knowledge)
+class WosQueries
+
+  # Default database 'WOK' is an umbrella for everything
+  DATABASE = 'WOK'.freeze
```

Comment:
Don't need a constant here, just put the string as the default in `initialize`.  That makes it part of the YARD.